datacats/cli/install.py

```diff
@@ -71,9 +73,13 @@ def install_all(environment, clean):
 
     for s in ['ckan'] + sorted(srcdirs):
         write('Installing ' + s)
-        environment.install_package_develop(s)
+        c_id = environment.install_package_develop(s)
+        if verbose and c_id:
+            print '\n'.join(container_logs(c_id, "all", True, None))
```

Comment:
we should be using the generator to get the terminal output as it arrives, not all at once after completion. Some of these installs are really slow.